onnxruntime/core/providers/cpu/nn/conv_transpose.cc

```diff
@@ -146,10 +143,10 @@ Status ConvTranspose<T>::DoConvTranspose(OpKernelContext* context, bool dynamic_
         // Col2im
         math::Col2imNd<T, CPUMathUtil, StorageOrder::NCHW>(
             col_buffer_data,
-            output_shape.GetDims().data(),
-            col_buffer_shape.data(),
+            output_shape.GetDims().data() + 1,
+            p.input_shape.GetDims().data(),
+            kernel_dim,
             output_shape.Size(),
```

Comment:
Is output_shape.Size() here the same as Y_offset? The code code be simplified to just use that and then do a Slice(2) to get output_shape above and get rid of the output_shape[0] line. Then you just pass output_shape.GetDims().data() instead of +1.

Reply:
Your are right, they are same. Modified.